plugins/uniques.js

```diff
@@ -30,543 +33,878 @@ window.plugin.uniques.contentHTML = null;
-			var portal = markup[2][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalVisited(guid);
-		} else if(plext.plextType == 'SYSTEM_BROADCAST'
-		&& markup.length==3
-		&& markup[0][0] == 'PLAYER'
-		&& markup[0][1].plain == nick
-		&& markup[1][0] == 'TEXT'
-		&& markup[1][1].plain == ' captured '
-		&& markup[2][0] == 'PORTAL') {
-			// search for "x captured y"
-			var portal = markup[2][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalCaptured(guid);
-		} else if(plext.plextType == 'SYSTEM_BROADCAST'
-		&& markup.length==5
-		&& markup[0][0] == 'PLAYER'
-		&& markup[0][1].plain == nick
-		&& markup[1][0] == 'TEXT'
-		&& markup[1][1].plain == ' linked '
-		&& markup[2][0] == 'PORTAL'
-		&& markup[3][0] == 'TEXT'
-		&& markup[3][1].plain == ' to '
-		&& markup[4][0] == 'PORTAL') {
-			// search for "x linked y to z"
-			var portal = markup[2][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalVisited(guid);
-		} else if(plext.plextType == 'SYSTEM_NARROWCAST'
-		&& markup.length==6
-		&& markup[0][0] == 'TEXT'
-		&& markup[0][1].plain == 'Your '
-		&& markup[1][0] == 'TEXT'
-		&& markup[2][0] == 'TEXT'
-		&& markup[2][1].plain == ' Resonator on '
-		&& markup[3][0] == 'PORTAL'
-		&& markup[4][0] == 'TEXT'
-		&& markup[4][1].plain == ' was destroyed by '
-		&& markup[5][0] == 'PLAYER') {
-			// search for "Your Lx Resonator on y was destroyed by z"
-			var portal = markup[3][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalVisited(guid);
-		} else if(plext.plextType == 'SYSTEM_NARROWCAST'
-		&& markup.length==5
-		&& markup[0][0] == 'TEXT'
-		&& markup[0][1].plain == 'Your '
-		&& markup[1][0] == 'TEXT'
-		&& markup[2][0] == 'TEXT'
-		&& markup[2][1].plain == ' Resonator on '
-		&& markup[3][0] == 'PORTAL'
-		&& markup[4][0] == 'TEXT'
-		&& markup[4][1].plain == ' has decayed') {
-		    // search for "Your Lx Resonator on y has decayed"
-			var portal = markup[3][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalVisited(guid);
-		} else if(plext.plextType == 'SYSTEM_NARROWCAST'
-		&& markup.length==4
-		&& markup[0][0] == 'TEXT'
-		&& markup[0][1].plain == 'Your Portal '
-		&& markup[1][0] == 'PORTAL'
-		&& markup[2][0] == 'TEXT'
-		&& (markup[2][1].plain == ' neutralized by ' || markup[2][1].plain == ' is under attack by ')
-		&& markup[3][0] == 'PLAYER') {
-		    // search for "Your Portal x neutralized by y"
-		    // search for "Your Portal x is under attack by y"
-			var portal = markup[1][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalVisited(guid);
-		}
-	});
+  var nick = window.PLAYER.nickname;
```

Comment:
Because of the indentation changes here, I can't see where the old code is and where the lines you changed are. In interface it looks like one huge change for whole function

Reply:
The same applies to almost the entire file.

Reply:
I was previously asked to take care of 2-spaces indentation and not use tabs. So the code was re-formated in all sections/functions that received new or changed code.

The atomic commits should help to follow up on the changes.
Shall I provide a DIFF that indicate the changes?

Reply:
Nothing to change